modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -11,19 +11,19 @@ If you plan to create the subnets in AWS Local Zones, you must opt in to each zo
 .Prerequisites
 
 * You have installed the AWS CLI.
-* You have determined into which region you will deploy your {product-title} cluster.
+* You have determined an AWS opt-in region for where you want to deploy your {product-title} cluster.
```

Comment:
Updated numerous instances of "AWS region" to "AWS opt-in region" to align with AWS docs: https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/opt-in-regions.html